app.js

```diff
@@ -22,6 +18,14 @@ app.use(handleCustomErrors)
 app.use(handlePsqlErrors)
 app.use(handleServerErrors)
 
+function getApiDetails(req,res,next) {
+  return fs.readFile(`${__dirname}/endpoints.json`)
+  .then((rawEndpoints) => {
+    const endpoints = JSON.parse(rawEndpoints)
+    res.status(200).send({endpoints})
+  })
```

Comment:
While not on this ticket, you could just require in the json and send it straight back. No need to read to a string, parse back to a json and send it.